langchain4j/src/test/java/dev/langchain4j/memory/chat/SummarizedTokenWindowChatMemoryTest.java

```diff
@@ -0,0 +1,86 @@
+package dev.langchain4j.memory.chat;
+
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.data.message.SystemMessage;
+import dev.langchain4j.data.message.UserMessage;
+import dev.langchain4j.memory.chat.prompt.SummarizedSystemPrompt;
+import dev.langchain4j.model.Tokenizer;
+import dev.langchain4j.model.chat.ChatLanguageModel;
+import dev.langchain4j.model.input.Prompt;
+import dev.langchain4j.model.input.structured.StructuredPromptProcessor;
+import dev.langchain4j.model.openai.OpenAiChatModel;
+import dev.langchain4j.model.openai.OpenAiTokenizer;
+import dev.langchain4j.store.memory.chat.ChatMemoryStore;
+import dev.langchain4j.store.memory.chat.InMemoryChatMemoryStore;
+import org.junit.jupiter.api.Test;
+
+import static dev.langchain4j.model.openai.OpenAiModelName.GPT_3_5_TURBO;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+
+class SummarizedTokenWindowChatMemoryTest {
```

Comment:
Your tests are failing on pipeline. Proper configuration of IT test could be found in: `dev.langchain4j.service.AiServicesIT`